build/templates/job.template.yaml

```diff
@@ -39,12 +38,11 @@ spec:
             echo "Gather timed out."
             exit 0
           fi
-        image: {{ $element }}
-        name: gather-{{ $index }}
+        image: quay.io/openshift/origin-must-gather:latest
```

Comment:
Looking at the available images, https://quay.io/repository/openshift/origin-must-gather?tab=tags&tag=latest Do we want to template this out to use the version of the cluster instead of `:latest`

Reply:
This makes sense to me. I'll look into making this happen.